packages/api-client/tests/test-granules.js

```diff
@@ -258,7 +258,7 @@ test('listGranules calls the callback with the expected object if there is no qu
       httpMethod: 'GET',
       resource: '/{proxy+}',
       path: '/granules',
-      body: undefined
+      queryStringParameters: undefined
```

Comment:
Same comment as above: why include this property if we don't want it to defined?